### PR TITLE
added test registration link that accepts json

### PR DESCRIPTION
**Pull request by supercilium**

### CODE REVIEW

File: dummy/Popoeshnick/app.py [hunk 1]
```diff
@@ -120,7 +119,7 @@ def login():
         })
 
 
-@app.route('/api/user/registration/', methods=['GET','POST'])
+@app.route('/api/user/registration11/', methods=['GET','POST'])
```
Comment (shirolapov) [inline dummy/Popoeshnick/app.py]: 11? WTF?

Comment (shirolapov) [inline dummy/Popoeshnick/app.py]: Registration must be only POST request. GET is not secure.

File: dummy/Popoeshnick/app.py [hunk 2]
```diff
@@ -143,5 +142,45 @@ def registration():
                     'email': form.email.data})
 
 
+@app.route('/api/user/registration/', methods=['POST'])
+def test():
+
+    # TODO: refactor, optimize passing email and password values to validator if necessary/possible
+    req_data = request.get_json()
+
+    print('!!111!', req_data)
```
Comment (shirolapov) [inline dummy/Popoeshnick/app.py]: No print!!!

File: dummy/Popoeshnick/app.py [hunk 3]
```diff
@@ -143,5 +142,45 @@ def registration():
                     'email': form.email.data})
 
 
+@app.route('/api/user/registration/', methods=['POST'])
+def test():
```
Comment (shirolapov) [inline dummy/Popoeshnick/app.py]: Test methods must be only in 'test.py' file and must be unittest.

File: dummy/Popoeshnick/app.py (hunk 1 above)
Comment (ptrwn) [reply dummy/Popoeshnick/app.py]: Fixed in the latest commit, left only /registration/ that accepts JSON.

Comment (ptrwn) [reply dummy/Popoeshnick/app.py]: Fixed in the latest commit -- removed GET, left only POST for /registration/

File: dummy/Popoeshnick/app.py (hunk 2 above)
Comment (ptrwn) [reply dummy/Popoeshnick/app.py]: Fixed in the latest commit - removed all print statements.

**Comment by ptrwn** [comment]
> Keep just one method with registration. No prints in code. No tests functions. Registration and login only on POST method.

these comments fixed in the latest commit, closing this pull request

**Pull request merged**